### DISCUSSION TITLE
Pure Theme With Bug

### DISCUSSION THREAD

**Discussion by MarlusArthur** [libre-tube/LibreTube#3694] [General]
Black Theme doesn't seem to work properly

![IMG_20230501_204333](https://user-images.githubusercontent.com/15721471/235550461-b3f6ada9-dba1-4e17-a017-51e8bce91834.jpg)

![Screenshot_2023-05-01-20-42-39-000_com github libretube](https://user-images.githubusercontent.com/15721471/235550481-1be8b504-407b-49c2-91fb-4cc6eedec20d.jpg)

**Comment by Silther** [comment]
Pleaes use GitHub issues for bug reports.
